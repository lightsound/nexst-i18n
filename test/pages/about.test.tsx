/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import About from "src/pages/about.page";

jest.mock("react-i18next", () => {
  return {
    useTranslation: () => {
      return {
        t: (key: any) => {
          return key;
        },
      };
    },
  };
});

describe("About page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<About />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
