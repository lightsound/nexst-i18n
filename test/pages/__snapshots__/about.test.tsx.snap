// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<DocumentFragment>
  <div>
    About
  </div>
</DocumentFragment>
`;
